lisa/sut_orchestrator/azure/features.py

```diff
@@ -44,6 +48,8 @@ def _initialize_information(self, node: Node) -> None:
         node_context = get_node_context(node)
         self._vm_name = node_context.vm_name
         self._resource_group_name = node_context.resource_group_name
+        self._location = node_context.location
```

Comment:
How about save only `self._node_context`? It doesn't need to add one by one in future.

Reply:
as new finding, it should be self._azure_runbook